data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_15.xml.ep

```diff
@@ -69,8 +69,8 @@
       % if ($check_var->('SYSTEM_ROLE', 'kvm')) {
         % if ($check_var->('AMD', '1')) {
           % if ($check_var->('VIRT_SEV_ES_GUEST_INSTALL', '1')) {
-            <!-- Do not add mem_encrypt=on option on 15-SP6 KVM host due to bsc#1224107 -->
-            % if ($check_var->('VERSION', '15-SP6')) { 
+            <!-- Do not add mem_encrypt=on on kernel command line for amd-zen3-gpu-sut1 which is the only machine affected by bsc#1224107 -->
+            % if ($get_var->('WORKER_CLASS', 'EMPTY_WORKER_CLASS') =~ /amd-zen3-gpu-sut1/) { 
```

Comment:
I have not had this experience, just wonder if we run the test by passing `WORKER_CLASS=64bit-ipmi-amd-zen3`, and the test run on `amd-zen3-gpu-sut1`, would the kernel options still be correct?

Reply:
Yes. It works.